src/transactions/transactions.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Controller, Get, HttpException } from '@nestjs/common';
+import { SwyftTablesInfo } from 'src/typings/types';
+import { TransactionsService } from './transactions.service';
+
```

Comment:
For swagger purposes try adding "@ApiTags('Transactions')" 
you will also need to: 
import { ApiTags } from '@nestjs/swagger';